sql/migrations/20170507001337_roundsRewards.sql

```diff
@@ -0,0 +1,158 @@
+/*
+ * Delete table 'rounds_fees', related functions and triggers
+ * Create table 'rounds_exceptions', calculate rewards & populate it, set triggers
+ */
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS rounds_exceptions (
+	"round"          INT PRIMARY KEY,
+	"rewards_factor" INT NOT NULL,
+	"fees_factor"    INT NOT NULL,
+	"fees_bonus"     BIGINT NOT NULL
+);
+
+-- Add exception for round 27040
+INSERT INTO rounds_exceptions VALUES (27040, 2, 2, 10000000);
```

Comment:
This shouldn't be hard coded in this way. Testnet should not end up with mainnet exceptions